Rakefile

```diff
@@ -120,7 +120,7 @@ namespace :rpm do
   end
 end
 
-desc "Release a new version ('type' is either 'major', 'minor 'or 'patch')"
+desc "Release a new version ('type' is either 'major' (first number increase), 'minor' (middle number increase) or 'patch' (last number increase))"
```

Comment:
I think it would be helpful to add some information about when to increase which number. That actually would be more helpful than which number is increased.